tendrl/commons/objects/cluster/atoms/import_cluster/__init__.py

```diff
@@ -64,14 +65,34 @@ def run(self):
                         )
             # Check if minimum required version of underlying gluster
             # cluster met. If not fail the import task
-            cluster_ver = \
-                NS.tendrl_context.sds_version.split('.')
-            maj_ver = cluster_ver[0]
-            min_ver = re.findall(r'\d+', cluster_ver[1])[0]
+            cmd = subprocess.Popen(
+                'rpm -qa | grep glusterfs-server',
+                shell=True,
+                stdout=subprocess.PIPE,
+                stderr=subprocess.PIPE,
+            )
+            out, err = cmd.communicate()
+            if out in [None, ""] or err:
+                raise FlowExecutionFailedError(
+                    "Failed to detect underlying cluster version"
+                )
+            lines = out.split('\n')
+            build_no = None
+            req_build_no = None
+            ver_det = lines[0].split('glusterfs-server-')[-1].split('.')
+            maj_ver = ver_det[0]
+            min_ver = ver_det[1]
+            rel = ver_det[2]
+            if '-' in rel:
```

Comment:
Add a comment with sample of whatever the ideal output of `rpm -qa | grep glusterfs-server` is for future reference,

No one's gonna understand this after 2 months :)

Reply:
Ack